src/admin/engineHeader.php

```diff
@@ -8,17 +8,17 @@
 $localvars  = localvars::getInstance();
 $enginevars = enginevars::getInstance();
 
-recurseInsert("acl.php","php"); 
+recurseInsert("acl.php","php");
 recurseInsert("includes/vars.php","php");
 recurseInsert('includes/functions.php',"php");
 
 recurseInsert('engineIncludes.php');
 
-formBuilder::process();
-formBuilder::ajaxHandler();
+// formBuilder::process();
+// formBuilder::ajaxHandler();
```

Comment:
If you are getting duplicate inserts from this being here, there is very likely something else wrong else where.